contracts/SignerBondsUniswapV2.sol

```diff
@@ -75,6 +75,10 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
     // Determines the deadline in which the swap transaction has to be mined.
     // If that deadline is exceeded, transaction will be reverted.
     uint256 public swapDeadline = 20 minutes;
+    // Determines if the open auctions check should be performed. If true,
+    // swaps cannot be performed if open auctions count is bigger than zero.
+    // If false, open auctions are not taken into account.
+    bool public openAuctionsCheck = true;
```

Comment:
Maybe `revertIfAuctionOpen`? This way we are more explicit about what this flag represents, "check" is too abstract.

Reply:
https://github.com/keep-network/coverage-pools/pull/88/commits/98f863ff3d68d17cb151559844177a59119aed52